### PR TITLE
Fix #33 ?

### LINKED ISSUE #33: wrong parsed arch on casey/just
``` 
>> uname -a
Linux jpambrun-rancher 3.10.0-1160.83.1.el7.x86_64 #1 SMP Wed Jan 25 16:41:43 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

>> curl -s https://i.jpillora.com/casey/just! | bash
Installing casey/just 1.13.0 (linux/amd64).....
######################################################################## 100.0%
mv with sudo...
Installed at /usr/local/bin/just

>> file /usr/local/bin/just
/usr/local/bin/just: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, BuildID[sha1]=5dd31aa9ff55b6b00d45dc70e07bac6d376f90f9, not stripped
```

Should have got x86_64, but got aarch64.

### PR DESCRIPTION
**Pull request by jpambrun**
This could still be much better.

### CODE REVIEW

**Comment by jpambrun** [comment]
This is untested.

**Comment by jpillora** [comment]
Ah you have a PR - thank you :) will check it out soon

File: handler/strings.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ import (
 )
 
 var (
-	archRe     = regexp.MustCompile(`(arm64|arm|386|amd64|32|64)`)
+	archRe     = regexp.MustCompile(`(arm64|arm|386|amd64|32|64|x86_64|aarch64)`)
```
Comment (jpillora) [inline handler/strings.go:9]: can you plz move these to the front? otherwise it'll return "64"

**Comment by jpillora** [comment]
Just one minor change, otherwise LGTM

File: handler/strings.go (hunk 1 above)
Comment (jpambrun) [reply handler/strings.go:9]: I did try it here[1] beforehand it seemed to macht the longest, but I can change regardless.

[1] https://regexr.com/77j6n

Comment (jpillora) [reply handler/strings.go]: Oh interesting, TIL
will test in Go soon and confirm

**Comment by jpillora** [comment]
Merged, should go live soon via CI

thanks!

(heading off to sleep 🤞)

**Pull request merged**